back/transcendence_django/back_game/game_arena/player_manager.py

```diff
@@ -92,17 +96,23 @@ def did_player_give_up(self, user_id: int) -> bool:
         except KeyError:
             return False
 
-    def get_winner(self) -> str:
-        active_players = [
-            player for player in self.players.values() if player.is_active()
-        ]
-        if (
-            not active_players
-            or len(set(player.score for player in active_players)) == 1
-        ):
-            return ""
-        winner = max(active_players, key=lambda player: player.score)
-        return winner.player_name
+    def get_game_summary(self) -> dict[str, Any]:
+        winner = self.__get_winner()
+        return {
+            PLAYERS: [
+                {
+                    USER_ID: player.user_id,
+                    PLAYER_NAME: player.player_name,
```

Comment:
No need to save in DB (or send to the frontend) the player_name as it is subject to changes, the front should send a request with the user id to get the corresponding user's current username.